docs/changelog/123986.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 123986
+summary: Failure Store Access Authorization
```

Comment:
Eh, actually I'll remove this and turn this into a non-issue. There are follow ups to complete this and besides the failure store itself isn't available yet so it's not really an enhancement.